src/orders/views/OrderDetails/OrderDraftDetails/index.tsx

```diff
@@ -113,15 +127,15 @@ export const OrderDraftDetails: React.FC<OrderDraftDetailsProps> = ({
       return;
     }
 
-    const result = await orderDraftUpdate.mutate({
+    await orderDraftUpdate.mutate({
       id,
       input: {
         user,
         userEmail
       }
     });
 
-    if (result?.data?.draftOrderUpdate?.errors?.length) {
+    if (orderDraftUpdate?.opts?.data?.draftOrderUpdate?.errors?.length) {
```

Comment:
I'm not sure if `orderDraftUpdate` would update after calling `mutate`. Can you confirm it actually happens?